From aaf7e0101b920bcbf63d8fb56e0a79b7103280fc Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@google.com>
Date: Mon, 6 Jul 2020 13:53:57 +0200
Subject: [PATCH 720/830] ANDROID: GKI: drop CONFIG_UAPI_HEADER_TEST from
 configs
X-NVConfidentiality: public

This option is no longer able to be selected due to some dependancy
changes between 5.8-rc3 and 5.8-rc4, so defconfig does not show it
anymore.  Remove it from the config files for now so that the build
works properly.

Signed-off-by: Greg Kroah-Hartman <gregkh@google.com>
Change-Id: I5f8a214e5890c0172dc6ed4d0e015209b72393a4
---
 arch/arm64/configs/gki_defconfig | 1 -
 arch/x86/configs/gki_defconfig   | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index d8b405a..f4476e3 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -1,4 +1,3 @@
-CONFIG_UAPI_HEADER_TEST=y
 CONFIG_LOCALVERSION="-mainline"
 CONFIG_AUDIT=y
 CONFIG_NO_HZ=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index 6e8ae77..6546b7b 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -1,4 +1,3 @@
-CONFIG_UAPI_HEADER_TEST=y
 CONFIG_LOCALVERSION="-mainline"
 CONFIG_KERNEL_LZ4=y
 # CONFIG_USELIB is not set
-- 
2.7.4

